### PR TITLE
Make template tag getitem not fail if field does not exist

### PR DESCRIPTION
**Pull request by pc-coholic**
Tiny change for the `getitem` template tag to not come down crashing if the requested key is not in the dictionary.

For current usage this change should not have any adverse effects, for new developments this brings adds the possibility of chaining the `|getitem` with `|default`.

### CODE REVIEW

**Comment by raphaelm** [comment]
:joy: I did the same thing today, just did not commit it yet

**Comment by raphaelm** [comment]
However, I did ``value.get('itemname')`` which returns ``None`` instead of ``""``. I assume ``""`` could have some problems with chained getitem calls? Not sure though

**Comment by pc-coholic** [comment]
I guess we could absolutely shorten this to your solution - I'll gladly close this issue in its favour :-)

Just a word on equally expected results...

Right now, we also return `''` if the dict is None... I feel there might be an argument, that a failed result should always be `None` or ` `. Or not - it's not like anyone else beside us is using this ;-)
